cves/CVE-2017-9608.yml

```diff
@@ -344,7 +356,13 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    One mistake that lead to the introduction of the vulnerability would be a lack
+    of proper testing. There is no clear unit tests running over this specific
+    function. Without proper edge case testing vulnerabilities like this can be
```

Comment:
```suggestion
    function. Without proper edge case, testing vulnerabilities like this can be
```

Reply:
I think the comma should actually be after vulnerabilities.